homeassistant/components/homekit/util.py

```diff
@@ -65,6 +69,8 @@
 _LOGGER = logging.getLogger(__name__)
 
 MAX_PORT = 65535
+VALID_VIDEO_CODECS = ["libx264", "h264_omx", "copy"]
```

Comment:
`Codec 'h264_omx' is not recognized by FFmpeg.`

This codec isn't compiled into the ffmpeg that comes with `Home Assistant` images

I think we can leave it in but we need to add a warning in the docs

Reply:
@bdraco, yes, you need to have an FFMpeg special build for RPi to be able to use h264_omx.

I will add a warning and an explanation regarding each one of the codec options in the documentation.